CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+[PR #33](https://github.com/fivetran/dbt_sage_intacct/pull/33) includes the following updates:
```

Comment:
I decided not to add the version to avoid creating noise with whatever PR it is brought into.